crates/re_log_encoding/src/file_sink.rs

```diff
@@ -43,7 +43,9 @@ pub struct FileSink {
     join_handle: Option<std::thread::JoinHandle<()>>,
 
     /// Only used for diagnostics, not for access after `new()`.
-    path: PathBuf,
+    ///
+    /// `None` indicates stdout.
+    path: Option<PathBuf>,
```

Comment:
... you could also have used an enum that makes this more obvious than reinterpreting the meaning of `None`

Reply:
Eh, I'd have gone there if it was part of the API but it's just some private field for debugging; not worth it imo.